src/main/java/com/redhat/rhjmc/containerjfr/MainModule.java

```diff
@@ -89,6 +89,17 @@ public static Gson provideGson() {
     @Provides
     @Named(RECORDINGS_PATH)
     static Path provideSavedRecordingsPath() {
-        return Paths.get("/", "flightrecordings");
+        String ARCHIVE_PATH = null;
+        final Logger logger = provideLogger();
+        try {
+            ARCHIVE_PATH = System.getenv("ARCHIVE_PATH");
```

Comment:
Can you refactor this to use the `Environment` utility class, which is also available by dependency injection? The source for that class is in `-core`, so you can take a look and see what methods might be useful for you here.

Reply:
Also, what do you think of prefixing this new env var with `CONTAINER_JFR_`? That's how the other configuration variables are generally named.

Reply:
After using the Environment class I realized I needed way less code!